### ISSUE TITLE
質問】【第3章：p.172】3-7_PSPNet_training の scheduler.step() について

### ISSUE THREAD

**Issue by mitgrad2018** [YutaroOgawa/pytorch_advanced#104]
小川様

お世話になっております。

3-7_PSPNet_trainingのコードを実行すると、以下のようなWarningが表示されます。
Warningにもある通り、PyTorch 1.1.0以降では、schedule.step() が optimizer.step() より前にあると、1つ目の learning rate がスキップされてしまうため、optimizer.step() の後に schedule.step() を持ってくる必要があるようです。

UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. 

学習結果への影響は小さいと思いますが、せっかくなので修正できればと思っておりますが、いかがでしょうか。
